src/system/Zikula/Module/SearchModule/AbstractSearchable.php

```diff
@@ -13,6 +13,7 @@
 
 namespace Zikula\Module\SearchModule;
 
+use Symfony\Component\DependencyInjection\ContainerInterface;
```

Comment:
Now that I see the new Search implementation again, I think it would be better to change it to behave like described here: http://symfony.com/doc/current/components/dependency_injection/tags.html

Then, no abstract classes are needed, just a single search interface for all bundles and modules. (will add a longer description ASAP).